makefiles/libc/newlib.mk

```diff
@@ -67,7 +67,15 @@ ifeq (1,$(USE_NEWLIB_NANO))
   NEWLIB_NANO_INCLUDE_DIR ?= $(NEWLIB_INCLUDE_DIR)/newlib-nano
   # newlib-nano overrides newlib.h and its include dir should therefore go before
   # the regular system include dirs.
-  INCLUDES := -isystem $(NEWLIB_NANO_INCLUDE_DIR) $(INCLUDES)
+  ifeq (0,$(words $(wildcard $(NEWLIB_NANO_INCLUDE_DIR))))
+    # HACK until this is really fixed. It keeps the old build system behaviour
```

Comment:
could you add a `TODO:` here? It is used at several other places and easy to grep for.